liquibase-standard/src/main/java/liquibase/changelog/StandardChangeLogHistoryService.java

```diff
@@ -245,14 +245,12 @@ public void init() throws DatabaseException {
             }
 
             SqlStatement databaseChangeLogStatement = new SelectFromDatabaseChangeLogStatement(
-                    new SelectFromDatabaseChangeLogStatement.ByNotNullCheckSum(),
+                    new SelectFromDatabaseChangeLogStatement.ByCheckSumNotLike(ChecksumVersion.latest().getVersion()),
```

Comment:
@filipelautert Do you think this should be checking `IS NOT NULL AND MD5SUM LIKE '...`, in other words, is the `IS NOT NULL` part important, which existed in the old version?

Reply:
> @filipelautert Do you think this should be checking `IS NOT NULL AND MD5SUM LIKE '...`, in other words, is the `IS NOT NULL` part important, which existed in the old version?

@StevenMassaro That's true, good catch! After a clearChecksums everything will be null and new line 253 will get a different result than old line 254 .